bundles/org.openhab.binding.ism8/src/main/resources/ESH-INF/i18n/ism8_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
Can this file be removed?

Reply:
Yes. I assume this was added when creating the binding via the script. Is my understanding correct that this is only used in case localized texts are used?
I'll remove the file

Reply:
Yes, you can use this file to easily translate the Channel names and so on to German.